.vscode/settings.json

```diff
@@ -0,0 +1,2 @@
+{
```

Comment:
This seems unnecessary. I wonder if we should gitignore the whole `.vscode` directory.

Reply:
done.